src/index.json

```diff
@@ -56400,6 +56400,51 @@
                 "sha256Digest": "58074ff73a72a1a9c76d406a4b6988a7b714dfaa9259b92172a7b6f66680db31"
             }
         ],
+        "stack-hci-vm": [
+            {
+                "downloadUrl": "https://hybridaksstorage.z13.web.core.windows.net/SelfServiceVM/CLI/stack_hci_vm-0.1.2-py3-none-any.whl",
+                "filename": "stack_hci_vm-0.1.2-py3-none-any.whl",
+                "metadata": {
+                    "azext.isExperimental": true,
```

Comment:
The isExperimental is not used any more. Please use isPreview instead

Reply:
Done